src/ComponentTester.cpp

```diff
@@ -27,17 +27,19 @@ void ComponentTester::testPin(){
   }
   if(componentType.equalsIgnoreCase("PHOTORESISTOR")){
     for(int i = 1; i <= 10; i++){
-        /*Serial.print("test#");
-        Serial.print(i);
-        Serial.print(" ");
-        Serial.println(analogRead(pin));*/
+        if(Properties::DEBUGGING_ACTIVE) {
+          Serial.print("test# " + i + ": ");
```

Comment:
So this line of code actually breaks the program. I don't know why C++ does this, but you can't easily concatenate Strings natively in c++. They need to be separated into multiple Serial.print() commands.